app/resources/results.html.template

```diff
@@ -46,16 +46,20 @@
       // third party search API directly here, to workaround AJAX's same
       // origin policy, and for authorization.
       function perform_third_party_search(id) {
-        jQuery.ajax({
-            url: repo_url + '/third_party_search',
-            data: {
-                query: query,
-                lang: lang,
-                query_type: query_type,
-                search_engine_id: id
-            },
-            dataType: 'json'
-        }).done(function(r) {
+        fetch(repo_url + '/third_party_search', {
```

Comment:
I'm wondering if it is safe to use fetch() and ES6 syntax e.g., `=>` here because they are relatively new and not supported by all browsers:
https://caniuse.com/#feat=promises,fetch
https://caniuse.com/#search=ecmascript
and we currently do not use ES6 here (except for the new React code which is transpiled). It might be OK since they are supported by all modern browsers. But may be safer to avoid them and use good old XMLHTTPRequest and `function(){}`? What do you think?

One alternative is to put this (removing jQuery) on hold until we launch the new React UI. Then this kind of HTML templates are only used by Lite mode and AMP where JS is disabled. So we can simply remove all JS code in HTML templates.

Reply:
Good point, changed to XMLHTTPRequest.
I think it's worth doing the work to get off jQuery now, since we're not sure when the new UI will launch. Also maybe we can consider having JS in lite mode as long as it's not critical functionality (e.g., the expanding/collapsing sections could still be used in lite mode as long as they're expanded by default when there's no JS).